nixos/doc/manual/release-notes/rl-2003.xml

```diff
@@ -75,6 +75,24 @@ services.xserver.displayManager.defaultSession = "xfce+icewm";
 </programlisting>
     </para>
    </listitem>
+   <listitem>
+    <para>
+     The testing driver implementation in NixOS is now in Python <filename>make-test-python.nix</filename>.
+     This was done by Jacek Galowicz <link xlink:href="https://github.com/tfc">(@tfc)</link>, and with the
+     collaboration of Julian Stecklina <link xlink:href="https://github.com/blitz">(@blitz)</link> and
+     <link xlink:href="https://github.com/jtraue">@jtraue</link>. All documentation has been updated to use this
```

Comment:
her real name is Jana Traue

Reply:
Cool, she's okay with that being here even when it isn't public information on her GitHub profile?

Reply:
Yes, Julian and Jana are ok with it, i just confirmed with them.